ev3dev/display.py

```diff
@@ -267,3 +277,77 @@ def update(self):
             self.mmap[:] = self._img_to_rgb565_bytes()
         else:
             raise Exception("Not supported")
+
+    def image_filename(self, filename, clear_screen=True, x1=0, y1=0, x2=None, y2=None):
+
+        if clear_screen:
+            self.clear()
+
+        filename_im = Image.open(filename)
+
+        if x2 is not None and y2 is not None:
+            return self._img.paste(filename_im, (x1, y1, x2, y2))
+        else:
+            return self._img.paste(filename_im, (x1, y1))
+
+    def shapes_line(self, clear_screen=True, x1=0, y1=0, x2=0, y2=0, line_color='black', width=1):
+
+        if clear_screen:
+            self.clear()
+
+        return self.draw.line((x1, y1, x2, y2), fill=line_color, width=width)
+
+    def shapes_circle(self, clear_screen=True, x=50, y=50, radius=40, fill_color='black', outline_color='black'):
```

Comment:
Document color options and how to make it transparent

Reply:
How do you make it transparent?  We are way beyond my experience level with PIL :)

Regarding supported color names, PIL says they support `Common HTML colour names`, I could never find a definitive list of the color names they support so didn't attempt to include that in the doc string.

Reply:
oh, I don't know... I've never used it :laughing: My guess is to pass `None`.